### ISSUE TITLE
heteroscedasticity  issue

### ISSUE THREAD

**Issue by wanghaisheng** [fwolling/PPGraw#2]
heteroscedasticity is that the variance is not constant

recently  I have read a paper about ppg signal processing,after a bunch of test ,the author say data collected from some unknown brand smart watch filtered ppg signal for different person  is too heterogeneity so it is not usable.

do we need this in quality review of raw photoplethysmography (PPG) signals,
@fwolling

**Comment by fwolling** [comment]
Yes, that makes sense - would be interesting to make a study specifically focusing on that! @wanghaisheng

**Comment by wanghaisheng** [comment]
after dig a little ,it seems quite simple , we just cal variance between diff person
